### PR TITLE
Feature 5 langingpage 01

### PR DESCRIPTION
**Pull request by Robel-003**
Draft PR... Look and layout very similar to Belinda's closet page. If there's a certain style NSC Events would prefer, I can add per request. General things about PR; UI looking ok and basic routing with buttons working. Overall, page still incomplete.
- Depending on css styling, still have to fix css to match demo Brandon provided 
- Input validation working ok, still needs fixing
- Forgot password page still incomplete but routing to page is working 
- Sign-in and Sign-up page not yet registering user's info because db hasn't been created yet 

Any and all feedback is greatly appreciated, thanks!

### CODE REVIEW

**Review by taylorpapke — DISMISSED** [review]
I see some good results here! I went ahead and ran this on VSCode and only had to do a simple npm install and npm run dev to get the app running in a React tab. I am able to navigate to sign up page (and forgot password page) and back to sign in page via home button. I can submit a sign in given the correct input into the text fields. The only issue I see is that the text field text is the same color as the text field (white) and is not visible unless you highlight it. In my opinion, this is a minor fix that can be easily improved upon in a later iteration. Do we want to switch this from a draft to a regular PR?

File: .gitignore [hunk 1]
```diff
@@ -4,6 +4,8 @@
 /node_modules
 /.pnp
 .pnp.js
+package-lock.json
+package.json
```
Comment (bcko) [inline .gitignore:9]: I believe we want to check both  `package-lock.json` and `package.json` to our repository. This way we all share the same package related json files.

File: package.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
     "eslint": "8.43.0",
     "eslint-config-next": "13.4.7",
     "next": "13.4.7",
+    "next-auth": "^4.22.1",
```
Comment (bcko) [inline package.json:19]: Let's pin `next-auth` to a specific version

File: styles/Home.module.css [hunk 3]
```diff
@@ -0,0 +1,116 @@
+.container {
```
Comment (bcko) [inline styles/Home.module.css:1]: Let's have a team discussion on how we want to use CSS. There are couple of options
- CSS Module (this pull request)
- Tailwind
- CSS in JS ( multiple options such as styled-components, emotion, styled-jsx etc)

**Comment by bcko** [comment]
@Robel-003 the pull request looks good. Feel free to convert your pull request from draft to regular

File: .gitignore (hunk 1 above)
Comment (Robel-003) [reply .gitignore:9]: fixed.. will push up a refactored pr

File: package.json (hunk 2 above)
Comment (Robel-003) [reply package.json:19]: fixed.. will push up a refactored pr

**Review by taylorpapke — APPROVED** [review]
Tested this again and everything is still working properly.

**Review by brinkbrink — APPROVED** [review]

**Comment by brinkbrink** [comment]
Looks like this repo is still requiring 3 approving reviews for merging @Seiyaroo @brandondombrowsky @bcko

**Comment by bcko** [comment]
> Looks like this repo is still requiring 3 approving reviews for merging @Seiyaroo @brandondombrowsky @bcko

Thank you for letting me know. Updated the repository settings

**Review by bcko — DISMISSED** [review]

**Review by taylorpapke — APPROVED** [review]
LGTM

**Review by brinkbrink — DISMISSED** [review]

**Review by taylorpapke — APPROVED** [review]
Yes, I can see the text in the text boxes now for the sign in page. However, I cannot see the "login" text as it is white over light grey.
My only other thought on this PR is that we could've maybe broken it into several separate smaller PR's to help with getting it approved and merged in sizable chucks. Those issues could have been:
1. create basic landing page with login text boxes and sign in.
2. add link to forgot password page (and add home button to navbar).
3. add link to sign up page.

We have a fully fleshed out landing page which is great but every time there is a change to the PR it has to go through review and approval all over again. This will be a good topic this week as a team to decide how big (or small) we want our issues to be. A more detailed discussion on PR approval settings might be a good idea as well.

**Comment by Robel-003** [comment]
> Yes, I can see the text in the text boxes now for the sign in page. However, I cannot see the "login" text as it is white over light grey. My only other thought on this PR is that we could've maybe broken it into several separate smaller PR's to help with getting it approved and merged in sizable chucks. Those issues could have been:
> 
> 1. create basic landing page with login text boxes and sign in.
> 2. add link to forgot password page (and add home button to navbar).
> 3. add link to sign up page.
> 
> We have a fully fleshed out landing page which is great but every time there is a change to the PR it has to go through review and approval all over again. This will be a good topic this week as a team to decide how big (or small) we want our issues to be. A more detailed discussion on PR approval settings might be a good idea as well. Great job overall!


Thanks for the feedback. I understand, and I think that's partly my fault in not adding those incremental functionalities and submitting pr's for quick reviews. I did not know how in depth the first issue (UA page) was suppose to be and I should've followed up with questions regarding that.

**Comment by taylorpapke** [comment]
Just need 1 more approval on this one to get it merged in.

**Review by brandondombrowsky — APPROVED** [review]
I think this code looks pretty clean and I like the comments. LGTM

**Pull request merged**